core/src/test/scala/unit/kafka/log/LogCleanerTest.scala

```diff
@@ -2045,6 +2045,45 @@ class LogCleanerTest extends Logging {
     }
   }
 
+  @Test
+  def testMaxOverCleanerThreads(): Unit = {
+    val logCleaner =  new LogCleaner(new CleanerConfig(true),
+      logDirs = Array(TestUtils.tempDir(), TestUtils.tempDir()),
+      logs = new Pool[TopicPartition, UnifiedLog](),
+      logDirFailureChannel = new LogDirFailureChannel(1),
+      time = time)
+
+    val cleaners = logCleaner.cleaners
+
+    val cleaner1 = new logCleaner.CleanerThread(1)
+    cleaner1.lastStats = new CleanerStats(time)
+    cleaner1.lastStats.bufferUtilization = 0.75d
```

Comment:
Nit: `d` is not needed, `0.75` already a double. Same for a few more cases below.

Reply:
Thank you 😸 
I have removed the redundant `d` but still keep it for the integer.